src/components/Tabs.react.js

```diff
@@ -121,6 +121,96 @@ EnhancedTab.defaultProps = {
     },
 };
 
+const NextTab = ({
+    className,
+    style,
+    selectHandler,
+    value,
+    disabled,
+    disabled_style,
+    disabled_className,
+    mobile_breakpoint,
+    amountOfTabs,
+    colors,
+    vertical,
+    loading_state,
+    title,
+}) => {
+    let tabStyle = style;
+    if (disabled) {
+        tabStyle = {tabStyle, ...disabled_style};
+    }
+    let tabClassName = `tab ${className || ''}`;
+    if (disabled) {
+        tabClassName += ` tab--disabled ${disabled_className || ''}`;
+    }
+    return (
+        <div
+            data-dash-is-loading={
+                (loading_state && loading_state.is_loading) || undefined
+            }
+            className={tabClassName}
+            id="next-tab"
+            style={tabStyle}
+            onClick={() => {
+                if (!disabled) {
+                    selectHandler(value);
+                }
+            }}
+        >
+            <span>{title}</span>
+            <style jsx>{`
```

Comment:
🌴🐫 Share common styling for `EnhancedTab` / `NextTab`
https://github.com/plotly/dash-core-components/pull/786/files#diff-87a94af645034b5ac219cbbfb96a6295R64